python/tests/tsutil.py

```diff
@@ -972,7 +972,7 @@ def mean_descendants(ts, reference_sets):
     last_update = np.zeros(ts.num_nodes)
     total_span = np.zeros(ts.num_nodes)
 
-    def update_counts(edge, sign):
+    def update_counts(edge, left, sign):
```

Comment:
wow good catch

Reply:
This was an interesting case - the linter said `left` wasn't being used... but it was! Via the closure and the fact a `for` loop "leaks" its loop variable to the outer scope!

Reply:
Wow, good work linter.